### DISCUSSION TITLE
是否有大佬使用b2作为存储

### DISCUSSION THREAD

**Discussion by kkfive** [lsky-org/lsky-pro#384] [问答] [answered]
看到后台支持设置s3，因此想使用[b2](https://www.backblaze.com/zh_CN/cloud-storage.html)作为存储，但是始终无法成功上传。

**Comment by love2wind** [comment]
![image](https://user-images.githubusercontent.com/11996852/163517846-06b324c5-5e95-48f0-85bf-60ab586f7bf0.png)
确实无法成功上传

**Reply by gfob** [reply]
我也遇到这样的问题了，把存储桶设置为私有可以上传成功，但是生成的链接不能使用，显示401。然后把存储桶设置为公开，之前生成的链接可以正常显示，但是又无法上传成功了，非常的奇怪。

**Comment by 0xxb** [comment, marked as answer]
@kkfive @gfob 详见 #448
